app/helpers/application_helper.rb

```diff
@@ -10,4 +10,12 @@ def nav_link_to(title:, path:)
       end
     end
   end
+
+  def feature_image(page:)
+    if page.featured_image.attached?
+      tag.div(class: "page-img-container") do
+        image_tag(url_for(page.featured_image), class: "page-responsive-img", alt: page.featured_alt_text, title: page.title, width: "500px")
+      end
+    end
+  end
```

Comment:
I'd suggest:
```ruby
def feature_image(page:)
  return unless page.featured_image.attached?

  tag.div(class: "page-img-container") do
    image_tag(url_for(page.featured_image), class: "page-responsive-img", alt: page.featured_alt_text, title: page.title, width: "500px")
  end
end
```
Also - should the width be defined in CSS via `.page-responsive-img` rather than hard coded into the HTML.

Reply:
I'll add the guard - the width in the html is for the browser and not for the css (see https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/)

Reply:
Re. My comment about width: My understanding of best practice is that it is better to define an element's appearance (including width) via CSS rather than directly in HTML. So:

css: `.page-responsive-img { width: 500px }`
html: `<img class="page-responsive-img">...</img>`

Rather than:
html: `<img class="page-responsive-img" width="500px">...</img>`

That makes it easier to do things like change the width depending on the surrounding or window size.